components/topbar/user/src/index.js

```diff
@@ -156,16 +156,19 @@ export default function TopbarUser({
    * Render main navigation function.
    */
   const _renderNavMain = isToggleHidden => (
-    {icon, label: text, menu, arrowButtonIcon},
+    {icon, label: text, menu, arrowButtonIcon, onClick},
```

Comment:
wouldn't this serve the same purpose and save you from declaring a new variable (L162)?

```suggestion
    {icon, label: text, menu, arrowButtonIcon, onClick: handleToggleMenu},
```

Reply:
I think it is interesting to leave the function ready in case in the future you want to do some previous action from the component itself before executing the onClick method, isn't it?

Reply:
fair enough... don't think it's a big deal performance wise but I always try to keep those "just in case" lines of code off the code, when possible 😉

Reply:
It's just one line of code 😛